README.md

```diff
@@ -1,25 +1,222 @@
+Get started on using the Cell Census:
 
-### Reuse
+- [Quick start](#Quick-start).
+- [Documentation](https://chanzuckerberg.github.io/cell-census/index.html). *Under development.*
+- [Python tutorials](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks).
+- R tutorials. *Coming soon.*
 
-The contents of this Github repository are freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT). Data in the CZ Cell Census are available for re-use under the [CC-BY license](https://creativecommons.org/licenses/by/4.0/).
+## Motivation: Single-cell analysis at scale 
 
-### For More Help
+The **Cell Census** is a data object publicly hosted online and a convenience API to open it. The object is built using the [SOMA](https://github.com/single-cell-data/SOMA) API and data model via its implementation [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA). As such, the Cell Census has all the data capabilities offered by TileDB-SOMA including:
 
-Please file a issue on this repo, or contact us at <cellxgene@chanzuckerberg.com>
+- Cloud-based data storage and access.
+- Efficient access for larger-than-memory slices of data.
+- Data streaming for iterative/parallelizable  methods.
+- R and Python support.
+- Export to AnnData and Seurat.
 
-### Security
+## Cell Census data releases
 
-If you believe you have found a security issue, we would appreciate notification. Please send email to <security@chanzuckerberg.com>.
+Starting in  mid 2023, Cell Census long-term supported data builds will be released every 6 months and are guaranteed to be stored for public access for at least 5 years upon release. 
 
-### Code of Conduct
+In between long-term supported data build releases, weekly builds are released without any guarantee of permanence. 
+
+## Cell Census data organization
+
+The Cell Census follows a specific [data schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema_0.1.0.md). Briefly, the Cell Census is a collection of a variety of **SOMA** objects organized with the following hierarchy.
+
+<img src="./docs/cell_census_data_model.svg">
+
+
+## Quick start
+
+### Requirements
+
+The Cell Census requires a Linux or MacOS system with:
+
+- Python 3.7 to Python 3.10. Or R, supported versions TBD.
+- Recommended: >16 GB of memory.
+- Recommended: >5 Mbps internet connection. 
+
+### Documentation
+
+Reference documentation, data description, and tutorials can be accessed at the Cell Census doc-site. *Coming soon*. 
+
+### Python quick start
+
+#### Installation
+
+It is recommended to install the Cell Census and all of its dependencies in a new virtual environment via `pip`:
+
+```
+python -m venv ./venv
+source ./venv/bin/activate
+pip install -U cell-census
+```
+
+#### Usage examples
+
+Demonstration notebooks can be found [here](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks).
```

Comment:
Highly recommend reminding people how to use docstrings, ie.

```python
help(cell_census)
```

or
```python
help(cell_census.get_anndata)
```

etc.